src/App.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Counter from './counter/Counter';
+import Search from './search/Search';
+import GenreToggle from './toggle/GenreToggle';
+
+// 1. React.createElement
+const myFullName = React.createElement('h1', {}, 'Khakim Bakhramov');
+
+// 2. React.Component
+class Greeting extends React.Component {
+    render() {
+        return <h3>Hello React</h3>
+    }
+}
+
+// 3. React.PureComponent
+class PureGreeting extends React.PureComponent {
+    render() {
```

Comment:
The same for React.PureComponent, it would be nice to use the features of PureComponent, e.g., to have either state or props in the component

Inside the Counter component there is a display of the result on the screen, it can be used as a PureComponent